### ISSUE TITLE
jitsi-meet-turnserver: Log spammed with `incoming packet BINDING processed, success`

### ISSUE THREAD

**Issue by paulmenzel** [jitsi/jitsi-meet#8910]
### Description:

Lately, at least with *jitsi-meet-turnserver* 1.0.4860-1the log is spammed with

```
Mar 29 22:10:06.887610 jitsi turnserver[407]: 52584: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
Mar 29 22:10:07.193845 jitsi turnserver[407]: 52584: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
Mar 29 22:10:07.309966 jitsi turnserver[407]: 52585: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
Mar 29 22:10:07.615145 jitsi turnserver[407]: 52585: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
Mar 29 22:10:07.733731 jitsi turnserver[407]: 52585: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
Mar 29 22:10:08.038851 jitsi turnserver[407]: 52585: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
Mar 29 22:10:08.156067 jitsi turnserver[407]: 52585: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
Mar 29 22:10:08.457942 jitsi turnserver[407]: 52586: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
Mar 29 22:10:08.577971 jitsi turnserver[407]: 52586: session 006000000000000120: realm <jitsi.molgen.mpg.de> user <>: incoming packet BINDING processed, success
```

### Steps to reproduce:

1.  Install Jitsi Meet from `deb https://download.jitsi.org unstable/`
2.  I have no idea, what client causes the message.

### Expected behavior:

The success messages should not be logged.

### Actual behavior:

The log is spammed with success messages.

### Server information:

- Jitsi Meet version: 2.0.5713-1
- Operating System: Debian 10 (buster)

**Comment by damencho** [comment]
Please, when you have questions or problems use the community forum before opening new issues, thank you.

Is this causing an issue? The turnserver(coturn) we use is an external dependency and we just have [a default configuration ](https://github.com/jitsi/jitsi-meet/blob/master/doc/debian/jitsi-meet-turn/turnserver.conf) to configure it to be used with jitsi-meet. There is nothing we can do about it, other than tweaking its config. If you experience problems with it you can report it to the [cotrun project](https://github.com/coturn/coturn).

**Comment by paulmenzel** [comment]
Issue https://github.com/coturn/coturn/issues/626 is open upstream saying it’s a regression in coturn 4.5.2. But I am seeing this with *coturn* 4.5.1.1-1.1+deb10u2.

**Comment by damencho** [comment]
Thanks for the update. 
`@misi Thanks, you are right. Indeed, since coturn-4.5.2 (I recently upgraded) I don't have those BINDING log lines anymore.`
The BINDING message is actually fixed in 4.5.2.
